src/main/java/hw_3/Task_5.java

```diff
@@ -0,0 +1,47 @@
+package hw_3;
+
+import java.util.Scanner;
+
+public class Task_5 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int w = rightResult(scanner);
+        System.out.println(w);
+
+        int a = secondRightResult(scanner);
+        System.out.println(a);
+
+        scanner.close();
+    }
+
+    public static int rightResult(Scanner scanner) {
+        System.out.println("Enter char :");
+        String ch = scanner.next();
+        System.out.println("Enter two numbers :");
+        int one = scanner.nextInt();
```

Comment:
bad variable name. Come up with more understandable name. Lets name it as firstValue